src/seedu/addressbook/AddressBook.java

```diff
@@ -444,13 +444,13 @@ private static String getMessageForSuccessfulAddPerson(String[] addedPerson) {
 
     /**
      * Finds and lists all persons in address book whose name contains any of the argument keywords.
-     * Keyword matching is case sensitive.
+     * Keyword matching is case insensitive.
```

Comment:
Correct! Comments should be updated too.

However, you may have missed the help message for this command :-). Look for COMMAND_FIND_DESC